package.json

```diff
@@ -84,6 +84,7 @@
     "@babel/core": "^7.12.10",
     "@emotion/jest": "^11.8.0",
     "@parcel/transformer-typescript-tsc": "^2.4.1",
+    "@parcel/transformer-typescript-types": "^2.4.1",
```

Comment:
Do we need both of these?

Reply:
I think it was added by mistaken. Deleting now!